src/index.ts

```diff
@@ -0,0 +1,7 @@
+// Entry point for Typescript build
+import OktaSignIn from './widget/OktaSignIn';
+
+export default OktaSignIn;
+export { OktaSignIn };
```

Comment:
why would need to do both?
isn't `export default OktaSignIn` good enough?

Reply:
When we are using named exports for code, we will want to do both. This is getting a little bit ahead, I will remove it to avoid confusion, since `import { OktaSignIn }` would be a type-only import at this time.